hummingbird/ml/operator_converters/_tree_commons.py

```diff
@@ -131,7 +131,7 @@ def get_tree_implementation_by_config_or_depth(extra_config, max_depth, low=3, h
     """
     if constants.TREE_IMPLEMENTATION not in extra_config:
         if max_depth is not None and max_depth <= low:
-            return TreeImpl.gemm
+            return TreeImpl.perf_tree_trav
```

Comment:
why this change?

Reply:
Ok this is a typo from testing. Let me roll this back